src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -25,18 +25,18 @@ public class AddCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_STUDENTID + "STUDENTID "
+            + "[" + PREFIX_ATTENDANCE_RECORD + "Attendance]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_STUDENTID + "A0255333B "
+            + PREFIX_ATTENDANCE_RECORD + "friends "
+            + PREFIX_ATTENDANCE_RECORD + "owesMoney";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New student added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This student already exists in the class";
```

Comment:
should class be class book instead?

Reply:
I think class is fine, since it is more intuitive than a classbook